.drone/drone.jsonnet

```diff
@@ -537,18 +539,93 @@ local manifest_ecr(apps, archs) = pipeline('manifest-ecr') {
       event: ['pull_request', 'tag'],
     },
     image_pull_secrets: [pull_secret.name],
+    volumes+: [
+      {
+        name: 'cgroup',
+        host: {
+          path: '/sys/fs/cgroup',
+        },
+      },
+      {
+        name: 'docker',
+        host: {
+          path: '/var/run/docker.sock',
+        },
+      },
+    ],
+    // Launch docker images with systemd
+    services: [
+      {
+        name: 'systemd-debian',
+        image: 'jrei/systemd-debian:12',
+        volumes: [
+          {
+            name: 'cgroup',
+            path: '/sys/fs/cgroup',
+          },
+        ],
+        privileged: true,
+      },
+      {
+        name: 'systemd-centos',
+        image: 'jrei/systemd-centos:8',
+        volumes: [
+          {
+            name: 'cgroup',
+            path: '/sys/fs/cgroup',
+          },
+        ],
+        privileged: true,
+      },
+    ],
+    // Package and test the packages
     steps: [
-      run(
-        'test packaging',
-        commands=['make BUILD_IN_CONTAINER=false packages']
-      ) { when: { event: ['pull_request'] } },
-      run(
-        'publish',
-        commands=['make BUILD_IN_CONTAINER=false publish'],
-        env={
-          GITHUB_TOKEN: { from_secret: github_secret.name },
-        }
-      ) { when: { event: ['tag'] } },
+      run('write-key',
+          commands=['printf "%s" "$NFPM_SIGNING_KEY" > $NFPM_SIGNING_KEY_FILE'],
+          env={
+            NFPM_SIGNING_KEY: { from_secret: gpg_private_key.name },
+            NFPM_SIGNING_KEY_FILE: '/drone/src/private-key.key',
+          }),
+      run('test packaging',
+          commands=[
+            'go install github.com/google/go-jsonnet/cmd/jsonnet@latest',  // Test, install in build image instead
+            'make BUILD_IN_CONTAINER=false packages',
+          ],
+          env={
+            NFPM_PASSPHRASE: { from_secret: gpg_passphrase.name },
+            NFPM_SIGNING_KEY_FILE: '/drone/src/private-key.key',
+          }),
+      {
+        name: 'test deb package',
+        image: 'docker',
+        commands: ['./tools/packaging/verify-deb-install.sh'],
```

Comment:
This is awesome. I'd use some test library but am totally fine with a shell script for now.